tests/test_runner/test_checkpoint.py

```diff
@@ -146,8 +146,24 @@ def test_get_state_dict():
                         wrapped_model.module.conv.module.bias)
 
 
-def test_load_pavimodel_dist():
+def test_get_torchvision_models():
+    # Check if wrong model paths are given
+    # More details at https://github.com/open-mmlab/mmcv/pull/1668
+    import pkgutil
+    import torchvision
+    from importlib import import_module
+
```

Comment:
We can make a directory named `torchvision/models/detection`  and several python files.

Reply:
You mean, importing way should be different from that? I have no idea how I should fix it.
Could you tell me the way if you have any idea?

Reply:
Thank you.

Reply:
Hi, please clean up the commit history. The changed files are 48.

Reply:
I cleaned it. Thank you.